server/src/pages/leave-screenshots/server.js

```diff
@@ -4,7 +4,7 @@ const reactrender = require("../../reactrender");
 const { Shot } = require("../../servershot");
 const mozlog = require("../../logging").mozlog("leave-screenshots");
 
-const csrfProtection = csrf({cookie: true});
+const csrfProtection = csrf({cookie: {httpOnly: true, secure: true, sameSite: 'lax', key: '__Host-csrf'}});
```

Comment:
Well... I thought the code that splits out the csrf cookie would fit nicely into that module too. Unfortunately the other modules can't just import it from server due to circular dependencies, and none of the other modules seem right.